src/main/java/appeng/client/gui/me/common/PendingCraftingJobs.java

```diff
@@ -79,4 +86,12 @@ private static boolean hasNotificationEnablingItem(LocalPlayer player) {
 
     record PendingJob(UUID jobId, AEKey what, long requestedAmount, long remainingAmount) {
     }
+
+    public static Event<BiConsumer<List<ItemStack>, Player>> WirelessTerminalEvent = EventFactory.createArrayBacked(
```

Comment:
Should be final and have upper case name.

I think this should be a separate class, as I was also planning on using it for #49.

Reply:
I am not quite sure on the name yet, maybe something more like `SearchInventoryEvent`, since it isn't actually specific to Wireless Terminals